egress_to_assessment_env_services_sg.tf

```diff
@@ -0,0 +1,18 @@
+# Security group for assessment environment access
+resource "aws_security_group" "assessment_environment_services_access" {
+  vpc_id      = data.terraform_remote_state.networking.outputs.vpc.id
+  description = "Security group allowing access to assessment environment services"
```

Comment:
Nitpicking, but don't we usually end our descriptions with a period?

Reply:
See commit 6abc7c3.